TeamBuildingGame/Assets/Scripts/ObjectMover.cs

```diff
@@ -21,6 +25,14 @@ void Start()
         for (int i = 0; i < clientData.playerCount; i++)
         {
             players[i] = Instantiate(playerPrefab);
+            if (i < clientData.playerCount / 2)
+            {
+                players[i].GetComponent<MeshRenderer>().material = blueColor;
+            }
+            else
+            {
+                players[i].GetComponent<MeshRenderer>().material = redColor;
```

Comment:
Det her er vel bare noget midlertidigt for at vise farverne, ikke? :D

Reply:
Mere eller mindre 🤔 Det var lidt tænkt at det skulle fungere ligesom på serveren, der sætter den vist nok bare de to første spillere på et hold og de to næste på det andet. Det kunne måske fucke lidt hvis de forskellige spillere blev instantierede anderledes i unity 🤔

Reply:
Det er bare finno